test/local/account_routes.js

```diff
@@ -667,11 +667,24 @@ test('/account/login', function (t) {
         t.equal(mockMailer.sendVerifyLoginEmail.callCount, 0, 'mailer.sendVerifyLoginEmail was not called')
         t.notOk(response.verificationMethod, 'verificationMethod doesn\'t exist')
         t.notOk(response.verificationReason, 'verificationReason doesn\'t exist')
-      })
-      .finally(function () {
-        mockLog.close()
+      }).then(function () {
+        mockMailer.sendNewDeviceLoginNotification.reset()
       })
     }, t)
   }, t)
+
+  test('creating too many sessions causes an error to be logged', function (t) {
+    mockDB.sessions = function () {
+      return P.resolve(new Array(201))
```

Comment:
Normally I like to assert that both sides of the boundary condition behave as expected, so I'd add a test case for 200 items too. But also maybe I'm a bit too anal about these things so I'm happy if you choose to ignore this suggestion.